File: lat1dpbo_java/Mahasiswa.java
public class Mahasiswa{
    private String name;
    private String nim;
    private String prodigy;
    private String faculty;

    public Mahasiswa(){
        this.name = "";
        this.nim = "";
        this.prodigy = "";
        this.faculty = "";
    }

    public Mahasiswa(String name, String nim, String prodigy, String faculty){
        this.name = name;
        this.nim = nim;
        this.prodigy = prodigy;
        this.faculty = faculty;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getNim(){
        return this.nim;
    }

    public void setNim(String nim){
        this.nim = nim;
    }

    public String getProdigy(){
        return this.prodigy;
    }

    public void setProdigy(String prodigy){
        this.prodigy = prodigy;
    }

    public String getFaculty(){
        return this.faculty;
    }

    public void setFaculty(String faculty){
        this.faculty = faculty;
    }
}

File: lat1dpbo_cpp/Mahasiswa.cpp
#include <iostream>
#include <string>

using namespace std;

class Mahasiswa{
    private:
        string name;
        string nim;
        string prodigy;
        string faculty;
    
    public:
        Mahasiswa(){
            this->name = "";
            this->nim = "";
            this->prodigy = "";
            this->faculty = "";
        }

        Mahasiswa(string name, string nim, string prodigy, string faculty){
            this->name = name;
            this->nim = nim;
            this->prodigy = prodigy;
            this->faculty = faculty;
        }

        string get_name(){
            return this->name = name;
        }

        void set_name(string name){
            this->name = name;
        }

        string get_nim(){
            return this->nim = nim;
        }

        void set_nim(string nim){
            this->nim = nim;
        }

        string get_prodigy(){
            return this->prodigy = prodigy;
        }

        void set_prodigy(string prodigy){
            this->prodigy = prodigy;
        }

        string get_faculty(){
            return this->faculty = faculty;
        }

        void set_faculty(string faculty){
            this->faculty = faculty;
        }
};